models/commons/src/discounted-price/discounted-price-draft/fields-config.ts

```diff
@@ -0,0 +1,33 @@
+import { fake, TModelFieldsConfig } from '@commercetools-test-data/core';
+import { ProductDiscount } from '@commercetools-test-data/product-discount';
+import * as Money from '../../money';
+import { ReferenceDraft } from '../../reference';
+import type {
+  TDiscountedPriceDraftRest,
+  TDiscountedPriceDraftGraphql,
+} from '../types';
+
+// https://docs.commercetools.com/api/types#discountedpricedraft
+/**
+ * value - Reference to Money
+ * discount - Reference to a ProductDiscount Reference Object {typeId, id}
+ */
+const commonFieldsConfig = {
+  value: fake(() => Money.random()),
+};
+
+export const restFieldsConfig: TModelFieldsConfig<TDiscountedPriceDraftRest> = {
+  fields: {
+    ...commonFieldsConfig,
+    discount: fake(() => ReferenceDraft.presets.productDiscountReference()),
+  },
+};
+
+export const graphqlFieldsConfig: TModelFieldsConfig<TDiscountedPriceDraftGraphql> =
+  {
+    fields: {
+      ...commonFieldsConfig,
+      discount: fake(() => ProductDiscount.ProductDiscountDraft.random()),
```

Comment:
@CarlosCortizasCT I've doubts adding the discountRef in the draft I think they're not needed.

Reply:
Yes, `discountedRef` is not part of the `DiscountedProductPriceValueInput` entity, which represents the draft of this data model.
However, bear also in mind the `discount` prop is defined as a `ResourceIdentifierInput` rather than a `ProductDiscount`.